app/views/placeholder/home.html.erb

```diff
@@ -42,45 +36,24 @@
 </nav>
 
 <% if @showQuizWarning %>
-<div class="jumbotron text-center" style="background-color: #FFAC6D">
+<div class="jumbotron text-center" style="background-color: #FF000">
   <h5>Please take the quiz to see organization matches.</h5>
 </div>
 <% end %>
-<div class="container" style="margin-top:30px">
-
 
-  
-  <div class="row">
-    <div class="col-sm-4">
-      <div class="fakeimg">Organization Image</div>
-    </div>
-    <div class="col-sm-8">
-      <h4>Fake Organization</h4>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-    </div>
-  </div>
-  <br>
-  <div class="row">
-    <div class="col-sm-4">
-      <div class="fakeimg">Organization Image</div>
-    </div>
-    <div class="col-sm-8">
-      <h4>Fake Organization</h4>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-    </div>
-  </div>
-  <br>
-  <div class="row">
-    <div class="col-sm-4">
-      <div class="fakeimg">Organization Image</div>
-    </div>
-    <div class="col-sm-8">
-      <h4>Fake Organization</h4>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-    </div>
+<h3 class="matched"> Your matched organizations </h3>
+<div class="container" style="margin-top:30px">
+   <div class="row text-center grid">
+    <% @organizations.each do |org| %>
+  <figure class="snip1104 blue">

```

Comment:
We should have a discussion about possibly making partials for all of our components so that it's more modular? Idk just an idea!